test/e2e/testing-manifests/storage-csi/external-snapshotter/README.md

```diff
@@ -0,0 +1 @@
+The original file is (or will be) https://github.com/kubernetes-csi/external-snapshotter/blob/master/deploy/kubernetes/rbac.yaml
```

Comment:
https://github.com/kubernetes-csi/external-snapshotter/blob/master/deploy/kubernetes/rbac.yaml exists now, and your copy isn't identical to it.

Please remove the "(or will be)" and update the file.

Reply:
Done